_posts/en/2019-10-01-btse-exchange-operation.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Field Report: Exchange operation using Bitcoin technologies at BTSE'
+permalink: /en/btse-exchange-operation/
+name: 2019-10-??-btse-exchange-operation
+slug: 2019-10-??-btse-exchange-operation
+type: posts
+layout: post
+lang: en
+version: 1
+
+excerpt: >
+  A field report about how BTSE uses Bitcoin technologies to improve their
+  exchange operations.
+
+---
+
+{:.post-meta}
+*by [Carl Dong](https://twitter.com/carl_dong)<br>Engineer at Optech*
```

Comment:
To be consistent with the previous field report from AJ, this should go inside `_includes/articles/dong-btse-operation.md`

Reply:
Actually, from what I see this is how AJ did it as well?

https://github.com/bitcoinops/bitcoinops.github.io/blob/5297aea8dfa9174bced89aa111a88555391c3ef7/_posts/en/2018-07-30-xapo-consolidation.md#L16-L18